interfaces/Portalicious/src/app/components/metric-container/metric-container.component.html

```diff
@@ -1,13 +1,15 @@
-<dt class="px-2 pb-2 pt-1 txt-body-m">
-  {{ label() }}
-</dt>
-<dd class="px-2 font-bold txt-h-2">
-  @if (pending()) {
-    <app-skeleton-inline />
-  } @else {
-    {{ value() ?? '-' }}
-    @if (showAlert()) {
-      <i class="pi pi-exclamation-triangle text-red-500"></i>
+<div class="flex h-full flex-col">
+  <dt class="px-2 pb-2 pt-2 txt-body-m">
+    {{ metric().label }}
+  </dt>
+  <dd class="mt-auto border-b border-grey-300 px-2 pb-2 font-bold txt-h-2">
+    @if (!metric()) {
+      <app-skeleton-inline />
+    } @else {
+      {{ metric().value ?? '-' }}
+      @if (metric().showAlert) {
+        <i class="pi pi-exclamation-triangle text-red-500"></i>
```

Comment:
Can this have a textual equivalent?

Like:
```suggestion
        <i
          class="pi pi-exclamation-triangle text-red-500"
          title="!"
          aria-label="!"
        ></i>
```

(Not 100% sure this works as-is; Could also need `[attr.aria-label]`...)

Note: I've tried to figure out a good way to make this reusable; But it seems that this emphasize is (currently) only used for the property "Failed transfers", in technically explaining the emphasize could be redundant/confusing.

My alternative suggestion would be to make the `showAlert`-property return a text-string that would contain the actual alert/warning-text.
But in this current use-case, that would result in a redundant warning-text that is also the value's `label`, (i.e. "`Failed transfers`"), so I thought that wasn't  worth it (for now).